.gitignore

```diff
@@ -101,4 +101,5 @@ local.properties
 # Maven
 ######################
 /log/**
-/target/**
\ No newline at end of file
+/target/**
+/angular/demo/.vscode/
```

Comment:
Either globally ignore every .vscode folder or place the rule under /angular/demo/.gitinore